src/colorizer/ColorizeCanvas.ts

```diff
@@ -207,7 +220,30 @@ export default class ColorizeCanvas {
     return this.colorMapRangeMax;
   }
 
+  getTotalFrames(): number {
+    return this.totalFrames;
+  }
```

Comment:
Can this use `this.dataset.numberOfFrames` rather than duplicating state in this object?

Reply:
Yup, done! Added some default behavior if the dataset hasn't been loaded yet.